Bottom: 0f30ed22a8e534bf84ea0c70fd9650cb6cebd864
Top:    c27d867ad4aa3272562653403c7f0f0398a62b7c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-06-10 11:00:30 -0700

Refresh of use_acm

---

diff --git a/configure.ac b/configure.ac
index 35d79dd..3ce7a6e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -70,9 +70,6 @@ AC_CHECK_HEADER(valgrind/memcheck.h, [],
 fi
 fi
 
-AC_CHECK_HEADER(infiniband/acm.h, 
-	AC_DEFINE([USE_IB_ACM], 1, [setting IBACM support]), [])
-
 AC_CHECK_HEADER(infiniband/acm.h,
 	AC_CHECK_MEMBER(struct acm_msg.resolve_data, [],
 		AC_DEFINE(DEFINE_ACM_MSG, 1, [adding ACM message definition]),
